services/worker-manager/src/api.js

```diff
@@ -868,10 +868,7 @@ builder.declare({
     'page. You may limit this with the query-string parameter `limit`.',
   ].join('\n'),
 }, async function(req, res) {
-  const quarantined = req.query.quarantined || null;
```

Comment:
those are listed in the api definition above on line 848-849, if they are no longer used, should be dropped from declaration.
Are they still used?

Reply:
ah, full query is passed I see, nevermind